public/index.html

```diff
@@ -1,22 +1,18 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" dir="ltr" >
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content=""
-    />
-
+    <meta name="description" content="An app built with every body and heart in mind" />
+    <meta name="Angela Battillo" />
```

Comment:
I'm not certain why your name is here, is this an seo sort of thing?

Reply:
not seo, but I want all our names in this.